infra/main.bicep

```diff
@@ -130,6 +130,7 @@ param azureOpenAIVisionModelCapacity int = 10
   'openai_function'
   'semantic_kernel'
   'langchain'
+  'prompt_flow'
```

Comment:
do we also want to add this in :
1. LOCAL_DEPLOYMENT.md -> section: `ORCHESTRATION_STRATEGY`
2. README.md -> section: `About this repo`